client/my-sites/sidebar-unified/menu.jsx

```diff
@@ -40,6 +40,7 @@ export const MySitesSidebarUnifiedMenu = ( {
 	link,
 	selected,
 	sidebarCollapsed,
+	continueInCalypso,
```

Comment:
Shall we get this added to `propTypes` too for validation

Reply:
Done 923686b!